File: .vscode/C-Programs/Q5.c
//Problem Statement
//Write a C program to find the largest among three numbers.


#include <stdio.h>

int main() {
    int a, b, c;
    printf("Enter three numbers: ");
    scanf("%d %d %d", a, b, c); 

    if (a > b && a > c)
        printf("%d is the largest", a);
    else if (b > a && b > c)
        printf("%d is the largest", b);
    else
        printf("%d is the largest", c);

    return 0;
}
//Expected Output
    //Enter three numbers: 5 8 3
    //8 is the largest

File: .vscode/Java_Programs/Q5.java
//Problem Statement
//Write a Java program to reverse a given number.

import java.util.Scanner;

class ReverseNumber {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int num, reverse;
        System.out.println("Enter a number: ");
        num = sc.nextInt();

        while(num > 0) {
            int digit = num % 10;
            reverse = reverse * 10 + digit; 
            num /= 10;
        }

        System.out.println("Reversed number: " + reverse);
        sc.close();
    }
}

//Expected Output
        //Enter a number: 1234
        //Reversed number: 4321

